src/nyc_trees/apps/event/models.py

```diff
@@ -58,9 +61,16 @@ def has_space_available(self):
         return self.eventregistration_set.count() < self.max_attendees
 
     @property
-    def starting_soon(self):
-        # TODO: Implement
-        return True
+    def starting_soon_from_now(self):
+        return self.starting_soon(timezone.now())
+
+    def starting_soon(self, dt):
```

Comment:
Not required, it's somewhat conventional, a workaround to the fact that python modules don't support private declarations. It just states your intention not to use it elsewhere. A minor point.